test/test_fakeredis.py

```diff
@@ -719,6 +725,16 @@ def test_set_ex_using_timedelta(r):
     assert r.get('foo') == b'bar'
 
 
+def test_set_ex_overflow(r):
+    with pytest.raises(ResponseError):
+        r.set('foo', 'bar', ex=18446744073709561)
```

Comment:
I would repeat the `# Overflows long long in ms` comment (in case the other test is deleted some day).